sys/include/net/sock/dtls.h

```diff
@@ -496,6 +496,8 @@ extern "C" {
                                              hold credentials */
 #endif
 
+#define SOCK_DTLS_HANDSHAKE     (EXDEV) /**< Return value for a succesful handshake */
```

Comment:
```suggestion
#define SOCK_DTLS_HANDSHAKE     (EXDEV) /**< Return value for a successful handshake */
```

And probably vera++ will complain about line-length ;-)

Reply:
fixed :)